docker/docker/Dockerfile.linux.amd64

```diff
@@ -1,4 +1,6 @@
-FROM docker:18.09.0-dind
+FROM docker:19.03.5-dind
+
+ENV DOCKER_HOST=unix:///var/run/docker.sock
```

Comment:
Are you sure this is required? The command execution is not loading env variables, so I think this change can simply get reverted.

Reply:
No, it won't work if you revert this, docker client will try to connect on default value of `DOCKER_HOST` `tcp://docker:2375` otherwise

Reply:
This is from the entrypoint calling dockerd-entrypoint.sh which calls docker-entrypoint.sh which sets DOCKER_HOST when unset. I don't think there's anything needed from those entrypoint scripts for drone so we could change the entrypoint to:

```
ENTRYPOINT ["/bin/drone-docker"]
```

and then remove the `ENV DOCKER_HOST` for the same result. There's also no need for `--host` when starting dockerd.